reps_event_app/lib/ui/about.dart

```diff
@@ -15,20 +15,24 @@ class _AboutState extends State<About> {
   Widget build(BuildContext context) {
     appBarSize = MediaQuery.of(context).size.height / 3.5;
     return Scaffold(
-      body: Column(
-        children: <Widget>[
-          CustomAppBar(appBarTitle: Utils.about_page, appBarSize: appBarSize,),
-          Padding(
-            padding: const EdgeInsets.only(top:20.0, bottom: 16.0, left: 16.0, right: 16.0),
-            child: Text(
-              Utils.about_info,
-              style: TextStyle(
-                fontSize: 28,
-                fontFamily: Utils.zilla_slab,
+      //SingleChildScrollView is used to handle cases when screen size is small
+      //enough to have all elements (Text) or fit completely.
+      body: SingleChildScrollView(
```

Comment:
`SingleChildScrollView` is not making the screen responsive, it's making the screen scrollable. The user should not be able to Scroll in the About tab.
![Screenshot_1571290335](https://user-images.githubusercontent.com/34301187/66981253-d2158400-f0d0-11e9-8220-3c33cf708f5b.png)

The main issue that needs to be addressed is to hide the keyboard in the About tab. You can hide the keyboard by removing the focus when tabs are changed.